src/modules/FioAddress/components/ConnectWallets.js

```diff
@@ -200,22 +203,22 @@ class ConnectWallets extends React.Component<FioConnectWalletStateProps & OwnPro
   }
 }
 
-const rawStyles = {
+const getStyles = cacheStyles((theme: Theme) => ({
   view: {
     flex: 1
   },
   list: {
-    flex: 5,
-    backgroundColor: THEME.COLORS.WHITE
+    flex: 5
   },
   no_wallets_text: {
-    padding: scale(30),
-    fontSize: scale(22),
-    color: THEME.COLORS.GRAY_2,
+    padding: theme.rem(1.875),
```

Comment:
these should be more evenly divisible rem values such as 1.25 or 1.5

Reply:
ok